DEVELOPMENT.md

```diff
@@ -6,10 +6,10 @@
 * Download and install [Android Studio] (https://developer.android.com/studio/index.html)
 * Set up [Android Development Environment] (https://spring.io/guides/gs/android/)
 * [Fork this repo] (https://help.github.com/articles/fork-a-repo/)
-* Copy GoIVCodeStyle.xml into the codestyles folder under the IntelliJ home directory (create if it does not exist)
- *  Windows: `\.\config\codestyles\`
- *  OS X: `~/Library/Preferences//codestyles/`
- *  Linux: `~/./config/codestyles/`
+* Copy GoIVCodeStyle.xml into the codestyles folder under the Android Studio config/codestyles directory *(create if it does not exist)*
+ *  Windows: `\%USERPROFILE%\.{ANDROID_STUDIO_FOLDER}\config\codestyles\`
+ *  OS X: `~/Library/Preferences/{ANDROID_STUDIO_FOLDER}/codestyles/`
+ *  Linux: `./{ANDROID_STUDIO_FOLDER}/config/codestyles/`
```

Comment:
If the Windows one has a dot, I'd expect the Linux to have one too.
`{ANDROID_STUDIO_FOLDER}` should be something like `AndroidStudio2.1` right? Better say so...